### DISCUSSION TITLE
not well-formed (invalid token) error in SentinelWrapper.py

### DISCUSSION THREAD

**Discussion by fdelgadodelapuente** [isce-framework/isce2#503] [General]
Dear ISCE developers, 

I'm using the TOPS stack processor and I get the following error when unpacking some SLCs

[fdelgado@prs stackproc_201410_202205_coregistration]$ SentinelWrapper.py -c /home/fdelgado/insarproc/cordon_caulle/dsc/stackproc_201410_202205_coregistration/configs/config_slave_20211016
Completed Parsing the Configuration file
Functions to be executed:
['Function-1']
Running: Sentinel1_TOPS
['--dirname', '/home/fdelgado/cordon_caulle/s1_dsc/slcs/S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.zip /home/fdelgado/cordon_caulle/s1_dsc/slcs/S1B_IW_SLC__1SDV_20211016T095714_20211016T095741_029159_037AE2_CE3D.zip', '--swaths', '2', '--orbitdir', '/home/fdelgado/isce/esa/s1orb', '--outdir', '/home/fdelgado/insarproc/cordon_caulle/dsc/stackproc_201410_202205_coregistration/slaves/20211016', '--auxdir', '/home/fdelgado/isce/esa/s1orb', '--bbox', '-40.83 -39.26 -72.59 -71.6', '--pol', 'vv']
This is the Open Source version of ISCE.
Some of the workflows depend on a separate licensed package.
To obtain the licensed package, please make a request for ISCE
through the website: https://download.jpl.nasa.gov/ops/request/index.cfm.
Alternatively, if you are a member, or can become a member of WinSAR
you may be able to obtain access to a version of the licensed sofware at
https://winsar.unavco.org/software/isce
Input XML files:  ['/vsizip//home/fdelgado/cordon_caulle/s1_dsc/slcs/S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.zip/S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.SAFE/annotation/s1b-iw2-slc-vv-20211016t095739-20211016t095807-029159-037ae2-005.xml', '/vsizip//home/fdelgado/cordon_caulle/s1_dsc/slcs/S1B_IW_SLC__1SDV_20211016T095714_20211016T095741_029159_037AE2_CE3D.zip/S1B_IW_SLC__1SDV_20211016T095714_20211016T095741_029159_037AE2_CE3D.SAFE/annotation/s1b-iw2-slc-vv-20211016t095714-20211016t095739-029159-037ae2-005.xml']
Input TIFF files:  ['/vsizip//home/fdelgado/cordon_caulle/s1_dsc/slcs/S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.zip/S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.SAFE/measurement/s1b-iw2-slc-vv-20211016t095739-20211016t095807-029159-037ae2-005.tiff', '/vsizip//home/fdelgado/cordon_caulle/s1_dsc/slcs/S1B_IW_SLC__1SDV_20211016T095714_20211016T095741_029159_037AE2_CE3D.zip/S1B_IW_SLC__1SDV_20211016T095714_20211016T095741_029159_037AE2_CE3D.SAFE/measurement/s1b-iw2-slc-vv-20211016t095714-20211016t095739-029159-037ae2-005.tiff']
Manifest files:  ['/vsizip//home/fdelgado/cordon_caulle/s1_dsc/slcs/S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.zip/S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.SAFE/manifest.safe', '/vsizip//home/fdelgado/cordon_caulle/s1_dsc/slcs/S1B_IW_SLC__1SDV_20211016T095714_20211016T095741_029159_037AE2_CE3D.zip/S1B_IW_SLC__1SDV_20211016T095714_20211016T095741_029159_037AE2_CE3D.SAFE/manifest.safe']
Input XML files:  ['/vsizip//home/fdelgado/cordon_caulle/s1_dsc/slcs/S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.zip/S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.SAFE/annotation/s1b-iw2-slc-vv-20211016t095739-20211016t095807-029159-037ae2-005.xml']
Input TIFF files:  ['/vsizip//home/fdelgado/cordon_caulle/s1_dsc/slcs/S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.zip/S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.SAFE/measurement/s1b-iw2-slc-vv-20211016t095739-20211016t095807-029159-037ae2-005.tiff']
MANS:  /home/fdelgado/cordon_caulle/s1_dsc/slcs/S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.zip S1B_IW_SLC__1SDV_20211016T095739_20211016T095807_029159_037AE2_8A28.SAFE/manifest.safe
Setting IPF version to :  003.31
**not well-formed (invalid token): line 19, column 48**
[fdelgado@prs stackproc_201410_202205_coregistration]$ 

Apparently it is something related to a text formatting issue, but I can't find out how to solve it. The error only shows up with certain SLCs only.

Francisco

**Comment by fdelgadodelapuente** [comment]
problem solved. It was corrupted orbit files

**Reply by fgdiazm** [reply]
Hi Francisco,

I'm having the same problem. Could you please explain how you solved it?

Did you download the orbits separately?

Thanks

**Reply by fdelgadodelapuente** [reply]
Hi, 

Download again the orbit files and check that the files are not missing nor corrupted. 

Francisco,

**Comment by fgdiazm** [comment]
Francisco,

I noticed something: the orbit files need to be outside each subfolder.

S1A_OPER_AUX_POEORB_OPOD_20210304T105324_V20141026T225944_20141028T005944.EOF  S1B_OPER_AUX_POEORB_OPOD_20210305T035400_V20191018T225942_20191020T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210305T023255_V20141213T225944_20141215T005944.EOF  S1B_OPER_AUX_POEORB_OPOD_20210305T085342_V20191030T225942_20191101T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210307T003615_V20150506T225944_20150508T005944.EOF  S1B_OPER_AUX_POEORB_OPOD_20210305T135228_V20191111T225942_20191113T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210307T082748_V20150530T225944_20150601T005944.EOF  S1B_OPER_AUX_POEORB_OPOD_20210305T135438_V20170101T225942_20170103T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210307T234511_V20150717T225943_20150719T005943.EOF  S1B_OPER_AUX_POEORB_OPOD_20210305T183921_V20191123T225942_20191125T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210308T063815_V20150810T225943_20150812T005943.EOF  S1B_OPER_AUX_POEORB_OPOD_20210305T232413_V20170125T225942_20170127T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210308T141914_V20150903T225943_20150905T005943.EOF  S1B_OPER_AUX_POEORB_OPOD_20210305T232718_V20191205T225942_20191207T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210308T215538_V20150927T225943_20150929T005943.EOF  S1B_OPER_AUX_POEORB_OPOD_20210306T041546_V20191217T225942_20191219T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210309T053936_V20151021T225943_20151023T005943.EOF  S1B_OPER_AUX_POEORB_OPOD_20210306T085829_V20170218T225942_20170220T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210309T130633_V20151114T225943_20151116T005943.EOF  S1B_OPER_AUX_POEORB_OPOD_20210306T090105_V20191229T225942_20191231T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210309T202649_V20151208T225943_20151210T005943.EOF  S1B_OPER_AUX_POEORB_OPOD_20210306T180943_V20170314T225942_20170316T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210310T120341_V20160125T225943_20160127T005943.EOF  S1B_OPER_AUX_POEORB_OPOD_20210306T224518_V20170326T225942_20170328T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210310T194932_V20160218T225943_20160220T005943.EOF  S1B_OPER_AUX_POEORB_OPOD_20210307T032901_V20170407T225942_20170409T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210311T032455_V20160313T225943_20160315T005943.EOF  S1B_OPER_AUX_POEORB_OPOD_20210307T081333_V20170419T225942_20170421T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210311T112005_V20160406T225943_20160408T005943.EOF  S1B_OPER_AUX_POEORB_OPOD_20210307T125037_V20170501T225942_20170503T005942.EOF
S1A_OPER_AUX_POEORB_OPOD_20210311T190409_V20160430T225943_20160502T005943.EOF  S1B_OPER_AUX_POEORB_OPOD_20210307T171643_V20170513T225942_20170515T005942.EOF

That's how it worked for me.

**Reply by fdelgadodelapuente** [reply]
The orbit files can be wherever you want. You only need to specify the directory in the respective flag in the topsApp input files.
